modules/juce_audio_devices/juce_audio_devices.cpp

```diff
@@ -164,6 +164,12 @@ namespace juce
 
 //==============================================================================
 #elif JUCE_WINDOWS
+
+ #if JUCE_MINGW
+  } // namespace juce

```

Comment:
This would allow all the smart pointer and message window code that follows to leak into the global namespace.. (?)